scripts/deployerCaller.ts

```diff
@@ -1,6 +1,8 @@
 import * as zksync from 'zksync-web3'
```

Comment:
i think we can totally remove this file, was just a testing one

Reply:
You mean the whole file or the import?

Reply:
the deployerCaller file itself

Reply:
I'm using it to call the function deploy in fact after that get deployed. I'll migrate it to foundry later on.